src/module/server-group/ServerGroupCtrl.ts

```diff
@@ -53,6 +53,26 @@ export class ServerGroupCtrl {
     return this.isServerInState(server, SERVER_STATE_RUNNING);
   }
 
+  isAtLeastOneServerInRunningState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RUNNING);
+  }
+
+  isAtLeastOneServerInReloadRequiredState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RELOAD_REQUIRED);
+  }
+
+  isAtLeastOneServerInRestartRequiredState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_RESTART_REQUIRED);
+  }
+
+  isAtLeastOneServerInStoppedState(): boolean {
+    return this.isAtLeastOneServerInServerGroupInState(SERVER_STATE_STOPPED);
+  }
+
+  areAllServersInStoppedState(): boolean {
+    return this.areAllServersInServerGroupInState(SERVER_STATE_STOPPED);
```

Comment:
But wait @ryanemerson that's not the case! ``statusMapToArray`` method just takes the IMap and converts it to the array. There is no server call here, right?